chalice/awsclient.py

```diff
@@ -79,7 +80,7 @@ def create_function(self,
         # type: (...) -> str
         kwargs = {
             'FunctionName': function_name,
-            'Runtime': 'python2.7',
+            'Runtime': DEFAULT_LAMBDA_RUNTIME,
```

Comment:
>  Is that you may be unknowingly change the runtime if you omit it?

Yeah that's the main downside.  I can't think of a use case where, if I'm using python3, I'd silently want the runtime to change to 2.7.